components/tag-list/demo/tag-list.html

```diff
@@ -52,6 +52,27 @@ <h2>Basic Tag List</h2>
 				</d2l-tag-list>
 			</d2l-demo-snippet>
 
+			<h2>Clearable Tag List</h2>
+			<d2l-demo-snippet fullscreen>
+				<d2l-tag-list description="A bunch of example tags" clearable>
+					<d2l-tag-list-item text="Example Tag"></d2l-tag-list-item>
+					<d2l-tag-list-item text="Longer Example Tag - much much much much much longer"></d2l-tag-list-item>
+					<d2l-tag-list-item text="Another Example Tag"></d2l-tag-list-item>
+					<d2l-tag-list-item-mixin-consumer name="Custom Tag List Item"></d2l-tag-list-item-mixin-consumer>
+					<d2l-tag-list-item text="Example Tag 5"></d2l-tag-list-item>
+					<d2l-tag-list-item text="Example Tag 6"></d2l-tag-list-item>
+					<d2l-tag-list-item text="Example Tag 7"></d2l-tag-list-item>
+				</d2l-tag-list>
+			</d2l-demo-snippet>
+
 		</d2l-demo-page>
 	</body>
+	<script type="module">
```

Comment:
Did you run into trouble without the rAF and the `type="module"`?

This seems to work ok for me...
```javascript
<script>
	document.addEventListener('d2l-tag-list-item-cleared', e => {
		e.target.deleteItem();
		console.log(`d2l-tag-list-item-cleared event dispatched. Value: ${e.detail.value}, handleFocus: ${e.detail.handleFocus}`);
	});
</script>
```
I think `<script>...</script>` should be just inside the `</body>`.  I _think_ `head` and `body` are the only valid elements inside of `html`.

Reply:
Ah yup that works for me as well! I'd needed more time before when adding the listener to individual items but just adding the listener to the document it's fine.

Ah, I'll move that then. Thanks!